win32ss/user/user32/misc/imm.c

```diff
@@ -84,41 +89,19 @@ BOOL WINAPI IntInitializeImmEntryTable(VOID)
         } \
     } while (0);
 #include "immtable.h"
+    /* FIXME: Please implement all the IMM32 entries and return FALSE if proc was NULL. */
 
     return TRUE;
 }
 
-BOOL WINAPI InitializeImmEntryTable(VOID)
-{
-  bImmInitializing = TRUE;
-  return IntInitializeImmEntryTable();
-}
-
 BOOL WINAPI User32InitializeImmEntryTable(DWORD magic)
 {
     TRACE("Imm (%x)\n", magic);
 
     if (magic != IMM_INIT_MAGIC)
         return FALSE;
 
-    if (IMM_FN(ImmIsIME) != IMMSTUB_ImmIsIME)
-        return TRUE;
-
-    IntInitializeImmEntryTable();
-
-    if (ghImm32 == NULL && !bImmInitializing)
-    {
-       WCHAR ImmFile[MAX_PATH];
-       GetImmFileName(ImmFile, sizeof(ImmFile));
-       ghImm32 = LoadLibraryW(ImmFile);
-       if (ghImm32 == NULL)
-       {
-          ERR("Did not load! 2\n");
-          return FALSE;
-       } 
-    }
```

Comment:
The situation may not be that simple, see the big comment on the top.

Reply:
Partially reverted.